index.html

```diff
@@ -227,10 +223,22 @@ <h3>Skills</h3>
     </div>
 </div>
     </section>
-<!-- ─── About Me Section ─── -->    
+
+<!-- ─── Contact Me Section ─── -->    
+
 <section class="contact">
-    
+    <div class="end-text">
+        <h1>Contact me</h1>
```

Comment:
- [ ] Since this is not the major heading of your page and you have already used an `h1` tag, kindly consider changing this to other variants of the heading tag i.e. `h2` - `h6`. This is conventional and it avoids conflicting SEO algorithms from understanding your page. 👍